bin/_tag.sh

```diff
@@ -17,7 +17,7 @@ clean_head() {
 }
 
 named_tag() {
-    tag="$(git name-rev --tags --name-only $(git_sha_head))"
+    tag="$(git name-rev --tags --name-only "$(git_sha_head)")"
```

Comment:
Any reason the inner quotes need to be escaped in `validate_tag` but not here?

Reply:
Are you referring to the echo parts of `validate_tag`? If so, that is since we want to print the " in the output and were they not escaped they would declare the end of the string to be echoed, which I believe was not what is intended. I expect the intent was to have that output as something like:

```
Tag in "/tmp/myfile" does not match source tree:
foo ("/tmp/myfile")
bar (source)
```